docker/terra-node/entrypoint/npm_install.sh

```diff
@@ -0,0 +1,8 @@
+#/bin/sh -e
+# check if the mode modules folder exists and is populated.
+if [ -n "$(ls -A node_modules 2>/dev/null)" ]
+then
+  echo "node_modules installed"
```

Comment:
Lets make a bit more explicit. Something that will call out the local node_modules were used in the build. Maybe "local node_modules exists. Copying locale node_modules to docker container"